lib/LaTeXML/resources/XSLT/LaTeXML-tei.xsl

```diff
@@ -781,11 +781,11 @@
         </table>
     </xsl:template>
 
-    <xsl:template match="ltx:tabular/*">
+    <xsl:template match="ltx:tabular">
             <xsl:apply-templates select="@*|node()"/>
     </xsl:template>
 
-    <xsl:template match="ltx:tbody">
+    <xsl:template match="ltx:thead | ltx:tbody">
```

Comment:
Also `tfoot`, I think?

Reply:
Oh, yeah. Avoided PR dance and pushed directly in 58e6b1e2